ECMA2Yaml/IntellisenseFileGen/IntellisenseFileGenHelper.cs

```diff
@@ -55,6 +57,10 @@ public static void Start(string[] args)
                 // TODO: log error
                 return;
             }
+            if (string.IsNullOrEmpty(opt.LogFilePath))
+            {
+                _logFile = "log.txt";
+            }
             (_repoRootFolder, _fallbackRepoRootFolder) = ECMALoader.GetRepoRootBySubPath(_xmlDataFolder);
             _fileAccessor = new FileAccessor(_repoRootFolder, _fallbackRepoRootFolder);
```

Comment:
After ECMAStore.Build(), you can use OPSLogger.ErrorLogged to check if there's any error logged. If there's error, you can directly quit.

Reply:
updated.